sre-recipes/recipes/currencies_recipe/currencies_recipe.py

```diff
@@ -124,17 +117,17 @@ def _cause_multiple_choice(self):
         answer = input('Your answer: ')
         return answer
 
-    def _verify_broken_service(self):
+    def verify_broken_service(self):
         """Verifies the user found which service broke"""
-        answer = self._service_multiple_choice()
+        answer = self.service_multiple_choice()
         while answer.lower() != 'f':
             print('Incorrect. Please try again.')
             answer = input('Your answer: ')
         print('Correct! The frontend service is broken.')
 
-    def _verify_broken_cause(self):
+    def verify_broken_cause(self):
```

Comment:
As I mentioned in the meeting, would it make sense to refactor these into a common method that can be used across different recipes. Perhaps a method that takes in a prompt, a list of choices, and the correct answer.

This would help duplicate some of the logic and formatting logic so if you need to change it, it will only be in a single location.

Reply:
Great idea! I've refactored the generating multiple choice method into the base class to make things simpler!